packages/plugins/src/plugins/morpho/types.ts

```diff
@@ -19,3 +19,89 @@ export type ParsedUpdatedIndexes = {
     p2pIndex: string;
   };
 };
+
+export type MorphoAssetAPI = {
+  address: string;
+  decimals: number;
+  logoURI: string | null;
```

Comment:
```suggestion
  logoURI?: string;
```

or 

```suggestion
  logoURI: string | undefined;
```

We shouldn't use `null`.

Reply:
This is the response from the morpho api, they return null